cluster/gce/util.sh

```diff
@@ -119,7 +119,9 @@ fi
 NODE_INSTANCE_PREFIX=${NODE_INSTANCE_PREFIX:-"${INSTANCE_PREFIX}-minion"}
 WINDOWS_NODE_INSTANCE_PREFIX=${WINDOWS_NODE_INSTANCE_PREFIX:-"${INSTANCE_PREFIX}-windows-node"}
 
-NODE_TAGS="${NODE_TAG}"
+# NODE_TAG (expected to be) defined by caller
+# shellcheck disable=SC2153
+NODE_TAGS=$NODE_TAG
```

Comment:
agree with the disable, but why drop the quotes/braces?

Reply:
Quotes are superfluous at variable assignments.

```
echo 'foo bar' > foobar.txt
a=$(cat foobar.txt)
b="$(cat foobar.txt)"
```

will result in `a` and `b` having the same content, `foo bar`.

Reply:
The quotes and braces are stylistic to the standards used by the project. We have style guidelines to prevent personal preferences from littering the codebase with inconsistencies given the number of contributors involved.

Please reference the style guidelines for the project [here](https://github.com/kubernetes/community/blob/master/contributors/guide/coding-conventions.md#code-conventions)

And specifically these sections:

- https://google.github.io/styleguide/shellguide.html#quoting
- https://google.github.io/styleguide/shellguide.html#variable-expansion

```
Stay consistent with what you find; quote your variables; prefer "${var}" over "$var"
```

Reply:
Thank you for your attention, PR has been updated accordingly.